manubot/cite/curie/__init__.py

```diff
@@ -177,28 +147,29 @@ def standardize_curie(curie):
         raise ValueError(
             f"curie must be splittable by `:` and formatted like `prefix:accession`. Received {curie}"
         )
-    # do not yet understand capitalization
-    # https://github.com/identifiers-org/identifiers-org.github.io/issues/100
     prefix_lower = prefix.lower()
-    prefix_to_namespaces = get_prefix_to_namespace()
     try:
-        namespace = prefix_to_namespaces[prefix_lower]
+        registry = get_prefix_to_registry()[prefix_lower]
     except KeyError:
         raise ValueError(
-            f"prefix {prefix_lower} for {curie} is not a recognized prefix"
+            f"Prefix {prefix_lower} for {curie} is not a recognized prefix."
         )
-    return f"{namespace['curiePrefix']}:{accession}"
+    standard_prefix = registry.get("preferred_prefix") or registry["prefix"]
+    return f"{standard_prefix}:{accession}"
 
 
 def curie_to_url(curie):
     """
     `curie` should be in `prefix:accession` format
     """
     curie = standardize_curie(curie)
-    resolver_url = "https://identifiers.org"
-    return f"{resolver_url}/{curie}"
+    prefix, accession = curie.split(":", 1)
+    registry = get_prefix_to_registry()[prefix.lower()]
+    if "url" in registry:
```

Comment:
```suggestion
if "uri_format" in registry:
    return registry["uri_format"].replace("$1", accession)
```

Reply:
So the next daily build that writes to https://github.com/biopragmatics/bioregistry/blob/main/exports/registry/registry.json will change `url` to `uri_format`?

Reply:
It should have happened already, you've helped me catch a typo! New version coming momentarily ;)

then of course it will update properly on the nightly build

Reply:
Done as of https://github.com/biopragmatics/bioregistry/pull/244